estate/demo/estate_property.xml

```diff
@@ -0,0 +1,81 @@
+<odoo>
+    <record id="umbm_estate_01" model="estate_property">
+        <field name="name">Big Villa 3</field>
+        <field name="state">new</field>
+        <field name="description">A nice and big villa</field>
+        <field name="postcode">12345</field>
+        <field name="estate_property_type" ref="estate.umbm_01"/>
```

Comment:
Nit: if you provide an xml id without a . to ref, it looks inside the current module. So this would also work
```suggestion
        <field name="estate_property_type" ref="umbm_01"/>
```